app/src/main/java/de/westnordost/streetcomplete/tangram/MapFragment.java

```diff
@@ -559,7 +560,7 @@ private void saveMapState()
 	{
 		if(controller == null) return;
 
-		SharedPreferences.Editor editor = getActivity().getPreferences(Activity.MODE_PRIVATE).edit();
+		SharedPreferences.Editor editor = getActivity().getSharedPreferences(PREF_NAME, Activity.MODE_PRIVATE).edit();
```

Comment:
Ah ok,... that's the caveat. Hmm, it's a but ugly that other activities access the preferences of the map this way. But I don't have an idea right now to have a cleaner interface there.

Reply:
Mmh... me too...